src/components/Preferences/LocalIdentities.js

```diff
@@ -42,8 +45,6 @@ const LocalIdentities = ({
     return <EmptyLocalIdentities onImport={onImport} />
   }
 
-  const { identityEvents$ } = React.useContext(IdentityContext)
-  const { showLocalIdentityModal } = React.useContext(LocalIdentityModalContext)
```

Comment:
I wanted to avoid making this change until after https://github.com/aragon/aragon/pull/768 was merged, so we don't have too many conflicts :).

Reply:
Reverted :+1: